fn/result.go

```diff
@@ -81,11 +81,23 @@ func (r Result[T]) Option() Option[T] {
 	return r.Either.LeftToOption()
 }
 
+// Hush mutes the error value of the result.
+func (r Result[T]) Hush() Option[T] {
```

Comment:
Identical to the `Option` helper above?

Reply:
Yes this is an alias.

Reply:
If it's an alias shouldn't it call `Option` rather than duplicating its logic? 

What's the motivation for an alias for this vs just renaming `Option`?

Reply:
Because I intend to remove Option in a later (breaking) PR.

Reply:
Ah ok, definitely missing some context on the longer term plan for a v2 version of this. 

Maybe add a note that `Option` is deprecated as done for `WhenResult` then?